bsip-0076.md

```diff
@@ -0,0 +1,88 @@
+    BSIP: 0076
+    Title: Threshold for price feeds through voting
+    Authors: Abit More
+    Status: Draft
+    Type: Informational
+    Created: 2019-09-25
+
+# Abstract
+
+This BSIP defines a process to influence the pegging status of committee-owned smartcoins through poll voting.
+
+# Motivation
+
+In order to maintain the peg of the smartcoins, smartcoins holders now have the right to be always
+possible to buy collateral at a "fair" price. For committee-owned smartcoins, the collateral is BTS.
+However, when the BTS token is being dumped in Centralized Exchanges with fake volume and even fake supply,
+the value of the token is distorted, thus, forcing BTS token holders who have their BTS token trapped in
+debt positions as collateral to sell their collateral at the distorted price via margin calls or
+force-settlements is unfair.
+The same situation applies to non-committee-owned smart coins as well.
+
+Smartcoins are products of the BitShares platform, their rules are defined by the platform
+for the benefit of the platform and the BTS token holders.
+If a product brings more harm than benefit to the platform, its rules need to be optimized,
+under extreme conditions, the product needs to be halted.
```

Comment:
What when halting (faking price feeds) causes more damage than simply allowing a GS to trigger?

The platform itself doesn't rely on the state of bitassets, blocks will keep being produced after global settlement events.